### PR TITLE
Manual examples for UniformScaling

### PR DESCRIPTION
**Pull request by kshyatt**
I think it's nice to have examples of the behaviour the manual talks about for specific types like this.

### CODE REVIEW

File: doc/src/manual/linear-algebra.md [hunk 1]
```diff
@@ -221,6 +221,39 @@ operators are generic and match the other matrix in the binary operations [`+`](
 with the identity operator `I` is a noop (except for checking that the scaling factor is one)
 and therefore almost without overhead.
 
+To see the `UniformScaling` operator in action:
+
+```jldoctest
+julia> ustwo = UniformScaling(2);
```
Comment (fredrikekre) [inline doc/src/manual/linear-algebra.md]: Call it something else? Perhaps just `U` or something, took me a while to decipher what `ustwo` meant :)

**Review by fredrikekre — COMMENTED** [review]
Nice, could be worthwhile to add a concatenation example, since that is quite nice thing you can do with `UniformScaling`
```
julia> [I a]
2×4 Array{Int64,2}:
 1  0  1  2
 0  1  3  4
```

File: doc/src/manual/linear-algebra.md (hunk 1 above)
Comment (kshyatt) [reply doc/src/manual/linear-algebra.md]: sure, good idea :)

**Review by Sacha0 — APPROVED** [review]
lgtm! :) Perhaps adding an example using `I` explicitly (or accordingly modifying one of the examples you have already added) would be worthwhile?

**Comment by kshyatt** [comment]
OSX/Windows fails seem unrelated to me?

**Pull request merged**